### ISSUE TITLE
Bug: Section dropdown won't close

### ISSUE THREAD

**Issue by shinglyu** [ylc395/joplin-plugin-note-link-system#9]
Step to reproduce:
1. Type `@@ Note A` to create a link. Note A contains a few H1 sections.
<img width="563" alt="Screenshot 2022-02-23 at 15 15 58" src="https://user-images.githubusercontent.com/3250983/155337090-59f82fa1-5890-4a82-bb12-a7d39a977e10.png">

2.  The cursor is now right before the closing bracket. Type a space character.
<img width="563" alt="Screenshot 2022-02-23 at 15 16 05" src="https://user-images.githubusercontent.com/3250983/155337393-bbfed8d3-a66c-477d-9818-32b39ce38701.png">

3. Now press the right arrow key to move the cursor after the closing bracket

4. The sections dropdown won't close. If you click on it it will insert `#section-1` to the cursor location. The only way to close it is to restart Joplin
 
 If you click `ESC` or right arrow key right after step 1, the dropdown will close correctly. But if you type any character that is not part of the section titles (e.g. space, `x`, anything that is not a substring of `section-1` and `section-2`), the dropdown will not close.

**Comment by shinglyu** [comment]
@ylc395 thank you for fixing this! Any chance you can release a new version to npm soon?

**Comment by ylc395** [comment]
> @ylc395 thank you for fixing this! Any chance you can release a new version to npm soon?

see v0.5.3
